pkg/netdevice/vhostNet.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // VhostNetDeviceExist returns true if /dev/vhost-net exists
 func VhostNetDeviceExist() bool {
-	_, err := os.Stat("/dev/vhost-net");
+	_, err := os.Stat("/dev/vhost-net")
```

Comment:
> @zshi-redhat is this work intended to support the VF use case?

I wasn't intend to support VF use case as we already have the pciAddresses selector.
But looking at the `GetPfAddr` function, it returns VF pci address when PF doesn't exist, I think it could also work for VFs.

Reply:
This change was merged in https://github.com/intel/sriov-network-device-plugin/pull/263, I will rebase this PR.

Reply:
Done